project/src/pages/favorites-page/favorites-page.tsx

```diff
@@ -17,12 +24,8 @@ function FavoritesPage(): JSX.Element {
                     </a>
                   </div>
                 </div>
-                <div className="favorites__places">
-                  <Card />
-                  <Card />
-                </div>
+                <OfferList offers={offers} className={'favorites__places'} />
```

Comment:
Когда строку передаем пропсом то 
```suggestion
                <OfferList offers={offers} className="favorites__places" />
```